R/ParamHandle.R

```diff
@@ -6,14 +6,144 @@
 #'
 #' @return [\code{\link{ParamHandle}}].
 #' @family ParamHelpers
+#' @export
+
 ParamHandle = R6Class("ParamHandle",
   inherit = ParamBase, # FIXME: Are we sure? Yes!
   public = list(
-
-    root = NULL,
+    # member variables
+    id = NULL,
```

Comment:
Do we need the id? If it is always the id of the node it should be an active binding?

Reply:
currently if you do not specify that explicitly, it will always be the same id with ParamNode. Maybe we need that in the future